src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdProviderMetadataNames.cs

```diff
@@ -56,6 +58,60 @@ public static class OpenIdProviderMetadataNames
         public const string UserInfoEncryptionAlgValuesSupported = "userinfo_encryption_alg_values_supported";
         public const string UserInfoEncryptionEncValuesSupported = "userinfo_encryption_enc_values_supported";
         public const string UserInfoSigningAlgValuesSupported = "userinfo_signing_alg_values_supported";
+    }
 #pragma warning restore 1591
+
+    /// <summary>
+    /// OpenIdProviderConfiguration MetadataName - UTF8Bytes
+    /// http://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata
+    /// </summary>
+    internal static class OpenIdProviderMetadataUtf8Bytes
+    {
+        public static readonly byte[] AcrValuesSupported = Encoding.UTF8.GetBytes("acr_values_supported");
```

Comment:
same question, can we use ReadOnlySpan and u8 strings?

Reply:
Yep, that is what Stephen wants us to do.